.github/workflows/sync-docs-from-node.yml

```
@@ -25,879 +29,519 @@ on:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        
+      - name: Download merged synced files artifact
+        uses: actions/download-artifact@v4
+        continue-on-error: true
+        with:
+          name: synced-merged
+          path: artifacts/
+      
+      - name: Generate workflow summary
         run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2129:style:1:1: Consider using { cmd1; cmd2; } >> file instead of individual redirects

(shellcheck)